generators/php/sdk/src/endpoint/request/WrappedEndpointRequest.ts

```diff
@@ -176,9 +233,136 @@ export class WrappedEndpointRequest extends EndpointRequest {
             inlinedRequestBody: (_inlinedRequestBody) => {
                 return php.codeblock(`${this.getRequestParameterName()}`);
             },
-            fileUpload: () => undefined,
+            fileUpload: () => {
+                return php.codeblock(`$${this.context.getPropertyName(this.wrapper.bodyKey)}`);
+            },
             bytes: () => undefined,
             _other: () => undefined
         });
     }
+
+    private getRequestBodyCode(): php.CodeBlock | undefined {
+        return this.endpoint.requestBody?._visit({
+            inlinedRequestBody: () => undefined,
+            reference: () => undefined,
+            fileUpload: (fileUpload) => this.getFileUploadRequestBodyCode(fileUpload),
+            bytes: () => undefined,
+            _other: () => undefined
+        });
+    }
+
+    private getFileUploadRequestBodyCode(fileUpload: FileUploadRequest): php.CodeBlock {
+        return php.codeblock((writer) => {
+            writer.write(`$${this.context.getPropertyName(this.wrapper.bodyKey)} = `);
+            writer.writeNodeStatement(
+                php.instantiateClass({
+                    classReference: this.context.getMultipartFormDataClassReference(),
+                    arguments_: []
+                })
+            );
+
+            for (const property of fileUpload.properties) {
+                property._visit({
```

Comment:
Discussed offline - let's roll out a function for each of these visitors, preferring the `switch` approach.

Something like the following:
```suggestion
                switch (property.type) {
                    case "file":
                      this.writeFileProperty(...)
                    case "bodyProperty:
                      this.writeBodyProperty(...)
                }
```